meter.go

```diff
@@ -251,13 +228,13 @@ func Write() error {
 	return Default.Write()
 }
 
-// StartFileWriter writes all metrics to a clear file.
+// StartFileWriter starts a goroutine that will periodically writes metrics to a file.
```

Comment:
That will write (without `s`). I'd rather rewrite it as:

`StartFileWriter starts a goroutine that periodically writes metrics to a file.`

Reply:
fixed